packages/ketcher-core/src/utilities/KetcherLogger.ts

```diff
@@ -0,0 +1,77 @@
+export enum LogLevel {
+  ERROR = 0,
+  WARN = 1,
+  LOG = 2,
```

Comment:
Can we add a method `info` here? It can be called this way `console.info()`. This is just additional method for the same logging, but it will allow you to reduce number of logs by setting proper log level

Reply:
Done